app/src/main/kotlin/com/wire/android/ui/common/SearchBar.kt

```diff
@@ -47,8 +49,9 @@ fun SearchBarUI(placeholderText: String, modifier: Modifier = Modifier, onTextTy
         leadingIcon = {
             IconButton(modifier = Modifier.height(40.dp), onClick = {}) {
                 Icon(
-                    Icons.Filled.Search,
-                    stringResource(R.string.content_description_clear_content)
+                    painter = painterResource(id = R.drawable.ic_search_icon),
+                    contentDescription = stringResource(R.string.content_description_conversation_search_icon),
+                    tint = MaterialTheme.colorScheme.onBackground
```

Comment:
no need to do that if we follow the default design values for the IconButton, under the hood IconButton provides the correct color, if it does not suit your needs than   tint = MaterialTheme.colorScheme.onBackground would make sense, do not know your use case

Reply:
Thanks, I should have specified `MaterialTheme.wireColorScheme.onBackground`. Good catch!